src/schedule/2021-03-03-thinking-in-react-with-keith-elliott.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Thinking in React with Keith Elliott
+type: Talk
+date: 2021-03-03T10:00:00-08:00
+---
```

Comment:
Keith, all of this looks great — great job!

There's one more field that can go here that, so far, we have not documented (so it wouldn't have come up at all), and that's the `speakers` field.

Would you be willing to add the below snippet to your fields here?

```suggestion
speakers:
  - Keith Elliott
---
```